ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_facts.py

```diff
@@ -581,6 +581,13 @@ def platform_settings(self):
 
         return {}
 
+    @cached_property
```

Comment:
Try maybe to add this closer to the `vtep_ip` one so they are kind of close by.
Wondering at the same time if it would be overkill to set it in the `avdipaddressing` module rather than here. Thoughts @ClausHolbechArista ?

Reply:
there is no algorithm here, just 1:1 fetching of the input variable. We don't even need it under switch and in facts. The plugins reading this value could just read from hostvars directly.

Reply:
so @emilarista what @ClausHolbechArista is suggesting is to actually have this line:

```
return get(self._hostvars, "vtep_vvtep_ip")
```

in `ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/utils.py` and to avoid making it a fact at all.

Reply:
resolved in 9859d0df0b4de2417633512031101fae198e89e8